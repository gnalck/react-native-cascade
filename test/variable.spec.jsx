import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BasicComponent from './VariableComponent';

it('renders correctly', () => {
  const tree = renderer
    .create(<BasicComponent />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
