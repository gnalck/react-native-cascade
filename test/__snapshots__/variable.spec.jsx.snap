// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  className="text-large container"
  ellipsizeMode="tail"
  style={
    Object {
      "color": "red",
      "fontSize": 24,
    }
  }
>
  Hello
</Text>
`;
